packages/framework/src/Command/CronCommand.php

```diff
@@ -162,7 +189,10 @@ private function runCron(InputInterface $input, CronFacade $cronFacade, MutexFac
      */
     private function getCurrentRoundedTime()
     {
-        $time = new DateTime(null);
+        $cronTimezone = $this->parameterBag->get('shopsys.cron_timezone');
+        $cronTimezone = $cronTimezone ?? date_default_timezone_get();
```

Comment:
I would move it to something like `getDateTimeZone(): DateTimeZone`

Reply:
updated